### ISSUE TITLE
Update filter input key SCImageView?

### ISSUE THREAD

**Issue by Sneakr** [rFlex/SCRecorder#188]
Is there any way to apply UISlider to update the value for example on "inputBrightness" ? 

The only way ive managed to get it work is if i use "setNeedsDisplay" or add the image to the SCImageView after value is changed on uislider but this is not the way to approach since it soaks up the CPU and memory. 

The input set on UISlider , does update but the SCImageView doesnt update itself.

**Comment by rFlex** [comment]
That is the correct way of doing it. Current implementation does not send any event to the SCImageView when some of the properties in SCFilter changes, this should be changed probably. From what I remember, calling setNeedsDisplay ask the SCImageView to redraw immediately. If you call 10 times setNeedsDisplay on the same run loop it's going to redraw 10 times to view, it should be done only once at the end of the current running loop.

TL;DR: You are doing it right, it should be improved on SCRecorder's side.

**Comment by Sneakr** [comment]
Ah I see, 

You are awesome, thanks for your great support! And thanks for a great project! :)
